analysis_options.yaml

```diff
@@ -2,17 +2,19 @@
 # for details. All rights reserved. Use of this source code is governed by a
 # BSD-style license that can be found in the LICENSE file.
 
-include: package:pedantic/analysis_options.yaml
+include: package:lints/recommended.yaml
 
 analyzer:
+  exclude: ['**bindings.dart'] # Exclude all generated bindings.
```

Comment:
Can we generate `// ignore_for_file: unused_import, unused_local_variable` in the files instead? So that we know how precisely we are violating the new lints?

We should also explore if some of the lints can be easily satisfied by changing the generator slightly.

Reply:
> We should also explore if some of the lints can be easily satisfied by changing the generator slightly.

I am only seeing 2 lint errors -
1. lowerCamelCase => Happens because of how we name the function pointer => using suffix `_ptr`.
2. upperCamelCase => Happens because of how we resolve name conflicts => using suffix `_<int>`.

Should we remove underscore from the suffix, and use => `Ptr` and `<Int>` instead?

Reply:
Let's do that.

Of course people will still get errors if the C names themselves have underscores (for example `JSON_Version`), but at least they can then make it pass the lints by renaming them if they would want to.

Reply:
Done. (The changes have been highlighted in the top comment)

The native type (if exposed using `symbol-address`) is still generated as `Native_<func>` which is not camel case. But I think we can fix this with #226